src/integrations/utils.ts

```diff
@@ -64,7 +64,7 @@ export async function initializeIntegrations(
   ]);
 
   if (gtm) initializeTagManager(gtm);
-  inferEmailLoginFromURL(clientArg);
+  if (fb || stytch) inferEmailLoginFromURL(clientArg);
```

Comment:
I think the login code checks to see if the client exists, but I don't see anything wrong with this check